scripts/add-global-changelog.js

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
Minor nitpick but should we name this file update-global-changelog.js instead?

Reply:
Happy to!